src/box/applier.cc

```diff
@@ -161,8 +161,6 @@ static int
 applier_writer_f(va_list ap)
 {
 	struct applier *applier = va_arg(ap, struct applier *);
-	struct iostream io;
```

Comment:
I try to make changes gradually, step by step: first replace ev_io with iostream and use it in exactly the same way (e.g. create a new iostream in applier writer); then reuse iostream created earlier instead of creating a new one (something that can be done with iostream, but not with ev_io).